zerver/openapi/zulip.yaml

```diff
@@ -5493,6 +5493,12 @@ paths:
 
                                         **Changes**: New in Zulip 9.0 (feature level 246). Previously, the Zulip
                                         server could not be configured to enforce unique names.
+                                    send_channel_events_messages:
+                                      type: boolean
+                                      description: |
+                                        Indicates whether channel event messages are sent in this organization.
+
+                                        **Changes**: New in Zulip 12.0 (feature level ZF-faec0b).
```

Comment:
Changes entries should always explain what happened prior to the change. Here, I'd add a sentence:

> Previously, channel events were sent unconditionally.